src/python/dxpy/nextflow/nextflow_utils.py

```diff
@@ -62,50 +64,20 @@ def get_regional_options(region):
     return regional_options
 
 def get_nextflow_assets(region):
+    basepath = path.dirname(__file__)
     # The order of assets in the tuple is: nextaur, nextflow
-    prod_assets = {
-        "aws:ap-southeast-2": ("record-GP5Bf385XGYQv71K5V25Q520",
-                               "record-GKv2BV05Q46P03jk5pQvVyBp"),
-        "aws:eu-central-1": ("record-GP5Bjqj4b633kF765jB330F2",
-                             "record-GKv2B384q2YZFjVz3jFPj7Zq"),
-        "aws:eu-west-2": ("record-GP5BgFjJXG2z2qk35G48f0vB",
-                          "record-GKv2BK8Jp9k14y5Y0bbGkf65"),
-        "aws:us-east-1": ("record-GP5BV700z3VxZ9xq5Qy3v962",
-                          "record-GKv1xPQ06J10XzPb3fVBF8XB"),
-        "azure:westeurope": ("record-GP5BZY0BKYf82PqX5kz6p875",
-                             "record-GKv2FGjBPKPP9k0x5Kpzj1ZX"),
-        "azure:westus": ("record-GP5BvZ890PK82PqX5kz6p94b",
-                         "record-GKv2j109b91Kg31j5Kj7V0Qj"),
-        "aws:eu-west-2-g": ("record-GP5Bp72KQ7b12Kx45vjfxF04",
-                            "record-GKv2B2XKJ6xP03jk5pQvVyBQ")
-    }
-    stg_assets = {
-        "aws:ap-southeast-2": (
-            "record-GP5B00Q56Y24843ffZ0Y83Gb",
-            "record-GKv18k85z6Qx78xb9kGkVB3j"),
-        "aws:eu-central-1": (
-            "record-GP5B2V046Z6J9kG5Vx49k0Kf",
-            "record-GKv18Qj43XYfYk2YGGkbYFVK"),
-        "aws:eu-west-2": (
-            "record-GP5B1FQJ1pz6gB19bv9BYkPP",
-            "record-GKv18b0Jz9YV9vqx9j9GGq05"),
-        "aws:us-east-1": (
-            "record-GP59VV80zXP4843ffZ0Y7fv2",
-            "record-GKv0P7Q00Q82gyJjGG8y03xf"),
-        "azure:westeurope": (
-            "record-GP59y98BQxYqB75pQy66X9FV",
-            "record-GKv19PjBJgXfVjgBGGB1Qjy2"),
-        "azure:westus": (
-            "record-GP5B6g89QjBFB75pQy66XVzP",
-            "record-GKv19X098bVGV15j9gpy8q2z"),
-        "aws:eu-west-2-g": (
-            "record-GP5B3p2Kb446439821Qqj02X",
-            "record-GKv1892K6Gkk0K53GFj415XZ")
-    }
-
+    nextaur_assets = path.join(basepath, "nextaur_assets.json")
+    nextflow_assets = path.join(basepath, "nextflow_assets.json")
     try:
-        dxpy.describe(prod_assets[region][0], fields={})
-        return prod_assets[region]
+        with open(nextaur_assets, 'r') as nextaur_f, open(nextflow_assets, 'r') as nextflow_f:
```

Comment:
Can we use https://github.com/dnanexus/dx-toolkit/blob/b92005049a92a6260a61043cea0d9c13924c54f4/src/python/dxpy/__init__.py#L979
to figure out whether we are on prod or stag?

Reply:
I believe this works only if you are not in a running job, that's why I chose to do it this way.

Reply:
then how about using `dxpy.APISERVER_HOST` directly?

Reply:
That still does not return 'staging' or 'prod', just a proxy IP in case of running from a job.

I've discussed this with Ola and I dont believe we have a convincing way of figuring this out.

Reply:
Got it. But if we could use env `DX_AUTHSERVER_HOST` in job exec env we could possibly avoid the call. But that could just be a minor improvement.